src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -26,23 +26,23 @@ const styles = {
     flex: 1, padding: '0 2rem 2rem 2rem'
   },
   rightBox: {
-    flex: 'none', width: 350, backgroundColor: colors.grayBlue[5],
+    flex: 'none', width: 350, backgroundColor: colors.light(0.4),
```

Comment:
The right sidebar is much lighter and is almost the same shade as the main part of the Workspace dash:
<img width="1249" alt="Screen Shot 2019-05-17 at 7 58 57 AM" src="https://user-images.githubusercontent.com/2395211/57926640-cc91e480-7879-11e9-97ef-79c23c384fe4.png">